frontend/public/components/monitoring/receiver-forms/email-receiver-form.tsx

```diff
@@ -0,0 +1,265 @@
+                <input
+                  className="pf-c-form-control"
+                  type="text"
+                  aria-describedby="email-auth-username-help"
+                  name="emailAuthUsername"
+                  data-test-id="email-auth-username"
+                  value={formValues.emailAuthUsername}
+                  onChange={handleChange}
+                />
+              </div>
+            </div>
+            <div className="col-sm-6">
+              <div className="form-group">
+                <label className="control-label">Auth Password (Using LOGIN and PLAIN)</label>
+                <input
+                  className="pf-c-form-control"
+                  type="password"
+                  aria-describedby="email-auth-password-help"
+                  name="emailAuthPassword"
+                  data-test-id="email-auth-password"
+                  value={formValues.emailAuthPassword}
+                  onChange={handleChange}
+                />
+              </div>
+            </div>
+          </div>
+          <div className="row">
+            <div className="col-sm-6">
+              <div className="form-group">
+                <label className="control-label">Auth Identity (Using PLAIN)</label>
+                <input
+                  className="pf-c-form-control"
+                  type="text"
+                  aria-describedby="email-auth-identity-help"
+                  name="emailAuthIdentity"
+                  data-test-id="email-auth-identity"
+                  value={formValues.emailAuthIdentity}
+                  onChange={handleChange}
+                />
+              </div>
+            </div>
+            <div className="col-sm-6">
+              <div className="form-group">
+                <label className="control-label">Auth Secret (CRAM-MDS)</label>
+                <input
+                  className="pf-c-form-control"
+                  type="password"
+                  aria-describedby="email-auth-secret-help"
+                  name="emailAuthSecret"
+                  data-test-id="email-auth-secret"
+                  value={formValues.emailAuthSecret}
+                  onChange={handleChange}
+                />
+              </div>
+            </div>
+          </div>
+          <label className="co-no-bold">
+            <input
+              type="checkbox"
+              name="emailRequireTls"
+              onChange={(e) =>
+                handleChange({
+                  target: { name: 'emailRequireTls', value: e.target.checked },
+                })
+              }
+              checked={formValues.emailRequireTls}
+            />
+            &nbsp; Require TLS
+          </label>
+        </div>
+      </div>
+    </div>
+  );
+};
+
+export const getInitialValues = (globals, receiverConfig) => {
+  const getInitialValue = (propName: string) =>
+    _.get(receiverConfig, propName) || _.get(globals, `smtp_${propName}`) || '';
```

Comment:
I think we know that `globals` is always an object, so we can drop some of the `_.get(globals, ...)` here and in the other `getInitialValues` functions.

Reply:
The `smtp_...` field may not be defined in the globals object, I thought using `_.get` returns false as opposed to getting run time error of 'property not defined' if I used `globals.smtp_auth_identity` when `smtp_auth_identity` isn't defined.